rifle.py

```diff
@@ -88,6 +125,55 @@ def draw_target_tags(self, screen):
                         (enemy.position_[0] * self.DRAWING_SCALE_ - (width/2),
                         (enemy.position_[1] * self.DRAWING_SCALE_ + enemy.size_/2) + (self.target_tag_y_spacing_ * self.DRAWING_SCALE_)))
 
+    def draw_friendly_projectiles(self, screen):
+        for projectile in self.universe_.friendly_projectiles():
+            projectile_rect = pygame.Rect((projectile.position_[0]-projectile.size_/2) * self.DRAWING_SCALE_,
+                                     (projectile.position_[1]-projectile.size_/2) * self.DRAWING_SCALE_,
+                                     projectile.size_ * self.DRAWING_SCALE_,
+                                     projectile.size_ * self.DRAWING_SCALE_)
+            pygame.draw.rect(screen, self.projectile_color_, projectile_rect)
+
+        main_character = self.universe_.main_character_
+        main_character_rect = pygame.Rect((main_character.position_[0]-main_character.size_/2) * self.DRAWING_SCALE_,
+                                          (main_character.position_[1]-main_character.size_/2) * self.DRAWING_SCALE_,
+                                          main_character.size_ * self.DRAWING_SCALE_,
+                                          main_character.size_ * self.DRAWING_SCALE_)
+        pygame.draw.rect(screen, self.main_character_color_, main_character_rect)
+
+class RifleProjectile(GameObject): 
+
+    def __init__(self, initial_position, target_position):
+        self.ID_ = self.create_ID()
+        self.initial_position_ = initial_position
+        self.position_ = self.initial_position_
```

Comment:
As coding style, I know `self.initial_position_` and `initial_position` are the same, but in general you should use parameters where possible instead of doing a `self` call.